src/main/services/csfloat/csfloatSocket.ts

```diff
@@ -0,0 +1,566 @@
+
+    const allTradeOffers = [
+      ...(tradeOffers.sent || []),
+      ...(tradeOffers.received || []),
+    ];
+
+    for (const trade of tradesInPending) {
+      if (trade.state !== EStatusTradeCSFLOAT.PENDING) continue;
+
+      if (!trade.wait_for_cancel_ping) continue;
+
+      const tradeOffer = allTradeOffers.find(
+        (steamTradeOffer) => steamTradeOffer.id === trade.steam_offer.id
+      );
+
+      if (
+        tradeOffer &&
+        (tradeOffer.state === ETradeOfferStateCSFloat.Active ||
+          tradeOffer.state === ETradeOfferStateCSFloat.Accepted)
+      )
+        continue;
+
+      try {
+        await this._csFloatClient.pingCancelTrade({
+          trade_id: trade.id,
+          steam_id: this.steamIDBase64,
+        });
+      } catch (error) {
+        console.error(
+          `failed to send cancel ping for trade ${trade.id}`,
+          error
+        );
+      }
+    }
+  }
+
+  private async autoAcceptTrades(
+    tradesInQueue: ITradeFloat[],
+    tradeOffers: IGetTradeOffersResponse
+  ): Promise<void> {
+    if (tradesInQueue.length < 1) return;
+
+    const tradeOffersSent = tradeOffers.sent;
+    const itemsTradables = await this.getItemsTradables()
+
+    if (itemsTradables.length === 0) return;
+
+    for (const tradeInQueue of tradesInQueue) {
+      const hasMatchingItem = itemsTradables.some(
+        (item) => item.assetid === tradeInQueue.contract.item.asset_id
+      );
+
+      if (!hasMatchingItem) continue;
+      let alreadyHasOffer = false
+      let state: ETradeOfferStateCSFloat;
+
+      for (const tradeOffer of tradeOffersSent) {
+        state = tradeOffer.state
+        for (const item of tradeOffer.itemsToGive) {
+          if (item.assetid === tradeInQueue.contract.item.asset_id && state != ETradeOfferStateCSFloat.Declined && state != ETradeOfferStateCSFloat.Canceled) {
+            alreadyHasOffer = true
+          }
+        }
+      }
+
+      if (!alreadyHasOffer) {
+        try {
+          const saleAccepted = await this._csFloatClient
+            .acceptTradesInFloat(tradeInQueue.id)
+          if (saleAccepted) this.emit("notifyWindows", {
+            title: `CSFLOAT - Accepted Sale`,
+            body: `Item: ${tradeInQueue.contract.item.item_name}!`,
+          });
+        } catch (error) {
+          this.emit("notifyWindows", {
+            title: `CSFLOAT - can not accept sale!`,
+            body: `Item: ${tradeInQueue.contract.item.item_name}!`,
+          });
+          console.error(error)
+        }
+      }
+    }
+  }
+
+
+  private async sendOffer(
+    tradesInPending: ITradeFloat[],
+  ): Promise<void> {
+    if (tradesInPending.length === 0) return;
+    for (const tradePending of tradesInPending) {
+      if (tradePending.buyer.steam_id === this.steamIDBase64) return;
+      if (tradePending.steam_offer.state === ETradeOfferStateCSFloat.Accepted || tradePending.steam_offer.state === ETradeOfferStateCSFloat.Active) return;
```

Comment:
use `continue` instead of `return`. `return` stops the function, `continue` skip to the next iteration of the loop
```suggestion
      if (tradePending.buyer.steam_id === this.steamIDBase64) continue;
      if (tradePending.steam_offer.state === ETradeOfferStateCSFloat.Accepted || tradePending.steam_offer.state === ETradeOfferStateCSFloat.Active) continue;
```